.github/workflows/ci.yml

```diff
@@ -126,7 +126,6 @@ jobs:
         - 5432:5432
     env:
       TEST_DATABASE_URL: postgresql://postgres@localhost:5432/report_test
-      DATABASE_URL: postgresql://postgres@localhost:5432/report_test
```

Comment:
This looks slightly wonky on `main` currently: `ci.yml` sets `TEST_DATABASE_URL` and `DATABASE_URL` to the exact same value.